packages/dapp/src/components/Widgets/Widgets.tsx

```diff
@@ -12,28 +17,31 @@ import { Widget } from '../Widget';
 import { WidgetEvents } from './WidgetEvents';
 import { WidgetContainer } from './Widgets.style';
 
+export type StarterVariantType = 'buy' | WidgetSubvariant;
```

Comment:
We have kind of related interface here `packages/dapp/src/components/Widget/Widget.tsx`. Can we combine them for reusability?